### ISSUE TITLE
Setting to auto-pin/unpin Presentations in Grid View

### ISSUE THREAD

**Issue by Eagle-Man** [Fugiman/google-meet-grid-view#170]
**Is your feature request related to a problem? Please describe.**
When a person presents in a full meeting (30-50 people) and I am in grid view, the presentation appears in a box of similar size to the other video feeds, that is to say, quite small so that I cannot see it easily.

**Describe the solution you'd like**
A setting so that when a presentation begins, it automatically becomes pinned (or at least take the positioning of a pinned stream), which will make it large enough to read, and automatically unpinned when the presentation ends.

**Describe alternatives you've considered**
I would have to manually pin the presentation and then unpin it when it's complete.

**Additional context**
I recognize that not everyone might want the presentation to take up that much real estate, especially if they are the presenter, so this is why I ask for it to be a setting.

**Comment by Fugiman** [comment]
Presentations should already be automatically pinned. It should look like: 
<img width="1920" alt="gmgv-presentation" src="https://user-images.githubusercontent.com/545785/83188403-7b4f3e80-a0e4-11ea-9d74-c9aa8a11a4c0.png">

If this isn't what you see please share an image of the meeting and press `CTRL+SHIFT+J` and share what it shows.

**Comment by Eagle-Man** [comment]
Hmm...I'll have to try it again with my class on Monday and I'll get that information.

**Comment by MarcAnt01** [comment]
> Hmm...I'll have to try it again with my class on Monday and I'll get that information.

I have joined some meetings with more than 70 people and the presentation box has always been bigger than the other ones.

**Comment by Eagle-Man** [comment]
Okay, here is what I get.  I am using Grid View to view my class.  When someone presents (in this case, me from another device in Present mode), I see the presentation in a small, unpinned video as if it were another person.  See the first image below.  Sorry, I didn't notice that a Hangouts notification had popped up in the corner.

What I would expect is what happens after I manually pin the video, see the second image.  And then when the presentation is done, I have to manually unpin.  Doing a quick poll of my students, there are a few of them that say the presentation appears small for them as well, and others say it's large (best I can do with them briefly without getting technical).  They should all have the most updated official extension, as it's managed by the school, but now that I think about it, they may not all have Grid View enabled (but at least a few others saw it small as well).

Windows 7
Chrome Version 83.0.4103.61 (Official Build) (64-bit)
Official Grid View 1.34.0

Settings: Include myself, show participants without video, highlight speaker, enable by default, no screen capture mode, obscure grid when toolbar is showing, resize when chat is showing, video is mirrored, show presentation when "include self" is selected, no modding names.

Note that "show presentation when 'include self' is selected" shouldn't come into play here, as it's not "myself" (on desktop) that's presenting, it's my connected laptop (same GAccount), which selected Present instead of Join Now, that's presenting.

![GridViewPresent](https://user-images.githubusercontent.com/53746675/83444662-8fef4780-a400-11ea-9218-eac6b074271d.png)
![GridViewPresentPinned](https://user-images.githubusercontent.com/53746675/83444682-97aeec00-a400-11ea-8465-f1a77aadac2d.png)

**Comment by Fugiman** [comment]
It looks like you have a lot of other extensions installed. Can you list what they are? You can also try enabling grid view on an incognito window and loading the meeting in that incognito window to determine if another extension is causing the issue.

**Comment by Eagle-Man** [comment]
Extensions: Grid View, Google Meet Attendance, Nod, Screencastify, Zoom Scheduler (though it happened before I installed this one), Google Meet Push-to-Talk.

I did a quick test with incognito mode and I had the same issue.  But then, on a whim, I tried connecting as a participant instead of in Presentation mode, and it worked properly.  I've created a [video demonstration](https://drive.google.com/file/d/1ZBNlBr6AXCumgKJ3MwHuEuJJVfB3ROdu/view?usp=sharing).
